### ISSUE TITLE
chore(developer): kmldmlc - Ensure that DTD is used when verifying XML files 🙀

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#7181]
* Ensure that DTD is used when verifying XML files in kmldmlc

From #7177

**Comment by mcdurdin** [comment]
Should be easy right?

```                                                                           
nnnn  nnnnnnnn       ooooooooooo   ppppp   ppppppppp       eeeeeeeeeeee    
n:::nn::::::::nn   oo:::::::::::oo p::::ppp:::::::::p    ee::::::::::::ee  
n::::::::::::::nn o:::::::::::::::op:::::::::::::::::p  e::::::eeeee:::::ee
nn:::::::::::::::no:::::ooooo:::::opp::::::ppppp::::::pe::::::e     e:::::e
  n:::::nnnn:::::no::::o     o::::o p:::::p     p:::::pe:::::::eeeee::::::e
  n::::n    n::::no::::o     o::::o p:::::p     p:::::pe:::::::::::::::::e 
  n::::n    n::::no::::o     o::::o p:::::p     p:::::pe::::::eeeeeeeeeee  
  n::::n    n::::no::::o     o::::o p:::::p    p::::::pe:::::::e           
  n::::n    n::::no:::::ooooo:::::o p:::::ppppp:::::::pe::::::::e          
  n::::n    n::::no:::::::::::::::o p::::::::::::::::p  e::::::::eeeeeeee  
  n::::n    n::::n oo:::::::::::oo  p::::::::::::::pp    ee:::::::::::::e  
  nnnnnn    nnnnnn   ooooooooooo    p::::::pppppppp        eeeeeeeeeeeeee  
                                    p:::::p                                
                                    p:::::p                                
                                   p:::::::p                               
                                   p:::::::p                               
                                   p:::::::p                               
                                   ppppppppp                               
```

There just ain't no DTD validators available for node. At least not ones that are maintained, or don't have extravagant native library dependencies. Who'd wanna use a DTD in 2022? But even XSD is a mess of library dependencies.

So... our best option may be for us to write an equivalent (or tighter if that is possible) json schema and use jsonschema? @srl295?

**Comment by mcdurdin** [comment]
Fixed by #7184.
